artbotlib/pr_in_build.py

```diff
@@ -248,8 +248,15 @@ def build_from_commit(self, task_state):
             if response_data and count > 0:
                 self.logger.info('Found %s builds from commit %s', count, commit)
                 builds = response_data["results"]
-                build_ids = [x["build_0_id"] for x in builds]
-                return sorted(build_ids)
+                build_ids = [(x["build_0_id"], x["build_0_name"]) for x in builds]
+                self.logger.info(f' *** these are the build_ids {build_ids} ***')
+
+                builds = response_data["results"]
+                max_values = {}
```

Comment:
the name max_values doesn't make sense here, let's change it to something else

```suggestion
                build_for_image = {}
```

Reply:
Done